product_url/models/model.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, api
+
+
+class ProductTemplate(models.Model):
+    _name = "product.template"
+    _inherit = [
+        "abstract.url",
+        "product.template",
+    ]
+
+
+class UrlUrl(models.Model):
+    _inherit = "url.url"
+
+    @api.model
+    def _reference_models(self):
+        res = super(UrlUrl, self)._reference_models()
+
+        res += [("product.template", "Product"),
+                ('product.category', "Category")]
```

Comment:
Perso j'utilise ' pour les infos courtes
    et " pour les infos longues.
Tout le monde ne fait pas comme ça, mais tout le monde est constant, fix it